website/client/js/controllers/memberModalCtrl.js

```diff
@@ -1,8 +1,8 @@
 "use strict";
 
 habitrpg
-  .controller("MemberModalCtrl", ['$scope', '$rootScope', 'Members', 'Shared', '$http', 'Notification', 'Groups', 'Chat', '$controller', 'Stats',
-    function($scope, $rootScope, Members, Shared, $http, Notification, Groups, Chat, $controller, Stats) {
+  .controller("MemberModalCtrl", ['$scope', '$rootScope', 'Members', 'Shared', '$http', 'Notification', 'Groups', 'Chat', '$controller', 'Stats', 'Content', 'Costume',
+    function($scope, $rootScope, Members, Shared, $http, Notification, Groups, Chat, $controller, Stats, Content, Costume) {
```

Comment:
Well, you added it, but it doesn't look like you're using it.  I think it can be removed.  Perhaps it got overlooked in one of your refactors?